### PR TITLE
100.14 version number updates.

### PR DESCRIPTION
**Pull request by mhdostal**
Updates the version numbers to 100.14.0 and Xcode to 13.0

### CODE REVIEW

**Comment by mhdostal** [comment]
@dfeinzimer @zkline101 Once approved, this will be merged after the SDK is released, just prior to the Toolkit release.

**Review by dfeinzimer — APPROVED** [review]

**Review by zkline101 — APPROVED** [review]
Approving these changes. 

But I was testing the toolkit with a local copy of the U14 release and I see 6 valid warnings in the examples app and 2 valid warnings in the toolkit. Can these be taken care of before U14 release?

**Comment by zkline101** [comment]
Is this ready to be merged?

**Pull request merged**
